guides/doc-Project_API/topics/snip_note-python3-vs-python.adoc

```diff
@@ -0,0 +1,5 @@
+[NOTE]
+====
+The example requests below use `python3` to format the response from the {ProjectServer}.
+On RHEL 7 and some older systems, you must use `python` instead of `python3`.
```

Comment:
Foreman/Katello only runs on EL8+. I am OK adding this snippet; but I think we could also drop it.

Reply:
The point is that you can call the API from a different machine than Foreman/SmartProxy server. That machine can run any OS different from the underlying OS of Foreman/SmartProxy server.

But if Python 2 has been EOT for two years, I think it's safe to remove it.